api/api.py

```diff
@@ -388,6 +388,7 @@ def trigger_mode(trigger_mode: TriggerMode):
 def dc_resource(name: str,
                 trigger_mode: TriggerMode = TRIGGER_MODE_AUTO,
                 resource_deps: List[str] = [],
+                auto_init: bool = True,
```

Comment:
i think order matters, so this has to be after `labels`

Reply:
I was mirroring `k8s_resource` and `local_resource` ordering. So you're saying that newly added named args need to be added at the end of the list, for compatibility?

Reply:
yep, you did it correctly here: https://github.com/tilt-dev/tilt/blob/master/internal/tiltfile/docker_compose.go#L124

Reply:
(we've talked to Alan + the starlark folks in the past about having an arg Unpacker that only allows keyword arguments, so that order doesn't matter: https://github.com/google/starlark-go/issues/249)